test/project-member.e2e-spec.ts

```diff
@@ -159,6 +159,13 @@ describe('ProjectMember e2e', () => {
         }
       )
     ).rejects.toThrowError();
+
+    const newProjectMember = await createProjectMember(app, {
+      userId: member.id,
+      projectId: project.id,
+    });
+
+    expect(newProjectMember.id).toBeTruthy();
```

Comment:
Can you put this in another test? Tests should only test one thing.

Reply:
got it, done